src/resources/postcss/components/_datepicker.pcss

```diff
@@ -0,0 +1,135 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Datepicker
+ *
+ * ----------------------------------------------------------------------------- */
+
+.tribe-events {
+
+	.datepicker {
+		background-color: var(--color-background);
+		border: 1px solid var(--color-border-default);
+		box-shadow: var(--box-shadow-tooltip);
+		margin-top: var(--spacer-1);
+		padding: 0 var(--spacer-3) var(--spacer-3);
+
+		@media (--viewport-medium) {
+			margin-top: var(--spacer-3);
+		}
+
+		&:before,
+		&:after {
+			display: none;
+		}
+
+		td,
+		th {
+			height: initial;
+			width: initial;
+		}
+
+		.dow {
+			color: var(--color-text-primary);
+			font-size: var(--font-size-0);
+			font-weight: var(--font-weight-regular);
+			padding: initial;
+			padding-bottom: var(--spacer-2);
+		}
+
+		.datepicker-switch {
+			color: var(--color-text-primary);
+			font-size: var(--font-size-3);
+			font-weight: var(--font-weight-regular);
+		}
+
+		.datepicker-switch,
+		.prev, .next {
+			line-height: initial;
+			padding: initial;
+			padding: var(--spacer-5) 0;
+
+			&:hover,
+			&:focus,
+			&:active {
+				background-color: var(--color-background);
+			}
+		}
+
+		.prev,
+		.next {
+			text-indent: -999em;
```

Comment:
we won't need to do this, as we can define what markup we use for the prev and next arrows. but i can take care of that on my end, so no need to do anything here.

Reply:
With the current implementation we don't have control over the markup. Is your implementation ready?

Reply:
not yet, but don't worry about the arrows, i can tackle them in the JS.